change_problem.py

```diff
@@ -23,45 +23,63 @@
 matplotlib.use("agg")
 import matplotlib.pyplot as plt
 
+# TODO: Enter your token here
+token = 'YOUR-TOKEN-HERE'
+
 # Set the solver we're going to use
-from dwave.system.samplers import DWaveSampler
-from dwave.system.composites import EmbeddingComposite
+def set_sampler():
+    from dwave.system.samplers import DWaveSampler
+    from dwave.system.composites import EmbeddingComposite
+
+    sampler = EmbeddingComposite(DWaveSampler(endpoint='https://cloud.dwavesys.com/sapi/', token=token, solver={'qpu': True}))
+
+    return sampler
+
+def create_graph():
+    # Create empty graph
+    G = nx.Graph()
+
+    ## TODO:  Update graph to new problem graph
+
+    return G
 
-sampler = EmbeddingComposite(DWaveSampler(solver={'qpu': True}))
+def solve_problem(G, sampler):
 
-# Create empty graph
-G = nx.Graph()
+    ## TODO:  Update dwave-networkx function to new problem function
 
-# Add edges to graph - this also adds the nodes
+    return 
 
-## TODO:  Update graph to new problem graph
+## ------- Main program -------
+if __name__ == "__main__":
 
-# Find the maximum independent set, S
+    G = create_graph()
 
+    sampler = set_sampler()
 
-## TODO:  Update dwave-networkx function to new problem function
+    # Find the maximum independent set, S
+    S = solve_problem(G, sampler)
 
-# Print the solution for the user
-print('Minimum vertex cover size found is', len(S))
-print(S)
+    # Print the solution for the user
+    print('Minimum vertex cover size found is', len(S))
+    print(S)
 
-# Visualize the results
-k = G.subgraph(S)
-notS = list(set(G.nodes()) - set(S))
-othersubgraph = G.subgraph(notS)
-pos = nx.spring_layout(G)
-plt.figure()
+    # Visualize the results
+    k = G.subgraph(S)
```

Comment:
What is `k` and what is it suppose to represent in the context of the problem? Perhaps give the variable another name?

Maybe `subgraph1`? And then the other subgraph could be `subgraph2`? Otherwise, maybe `a_subgraph` and `other_subgraph`?